### ISSUE TITLE
Request to add a "Copy location to subsequent images" button in the upload wizard

### ISSUE THREAD

**Issue by ShashwatKedia** [commons-app/apps-android-commons#5668]
### What is the user problem or growth opportunity you want to see solved?

As all images of any specific upload will essentially depict the same item, it's imperative that their location would be same. Thus, a "Copy location to subsequent images" button, similar to the "Copy to subsequent media" button for description/caption would be a good feature to have.

### How do you know that this problem exists today? Why is this important?

I have experienced this issue myself while uploading images of the same item from different angles but without EXIF data. I have to go through an extra step to manually add the location to each of the images. 

### Who will benefit from it?

The general users of the app, who use the multi-upload feature of the app to upload images without EXIF data, will greatly benefit from it, as it will reduce the time taken to add a location to each image individually.

### Anything else you would like to add?

I would like to work on this enhancement if it's required by the community :)

**Comment by ShashwatKedia** [comment]
> We already have a mechanism to suggest the coordinates of a picture taken shortly before or after, and we remember the last selected geolocation, so I am not sure a dedicated UI element would justify its screen real estate...

Yeah, we do have a similar image coordinate dialog, but that pops up only when the one image in the previous set of images had EXIF data. If my current set of images do not have any EXIF data, and I add location for the first image, there is no mechanism to copy that to the other images :(

**Comment by nicolas-raoul** [comment]
It only takes two quick taps for each picture. 🙂

**Comment by ShashwatKedia** [comment]
> It only takes two quick taps for each picture. 🙂

Fair point. I just thought it would be good to add such an element to enhance the upload wizard further and fasten the process :)

**Comment by kjuanman** [comment]
Huge support to this enhancement. I have taken thousands photos in the Olympics and I'm uploading about 30 photos by match with my phone (I don't have my computer here). The two quick taps by photo are turned into 60 taps by match and are making me crazy.
